### DISCUSSION TITLE
uWebsockets as an option?

### DISCUSSION THREAD

**Discussion by kronos-999** [supabase/supabase#20326] [Questions] [answered]
Hello, we have a very performante uWebsockets implementation and want to use supabase. We are looking for ways to combine those two approaches/technologies as seamless as possible.... Do you guys (probably someone from the supabase can answer this best) know what we could do? Maybe some perf comparisons or scenarios where it could make sense to use them separately?

**Comment by awalias** [comment, marked as answer]
Hi @kronos-999 , to my knowledge we haven't tried uWebsockets with supabase, but would be interested in learning what you discover!

**Comment by worldpeaceenginelabs** [comment]
@kronos-999 
I'm interested in exactly the same. Any progress in finding or creating a supabase/uwebsockets combination?
